Documentation/Parameters/JSONPathParamSample.workbook

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
do you want to link this in the parameters.md doc? I don't see this referenced anywhere

Reply:
oh, the workbook content itself is not linked, not sure if it needs to be.  this is more a best practice kind of thing that we should be doing that all of the screenshots/samples we use should have a sample workbook here in docs so someone can reproduce it.